tests/kryoptic.nss-init.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -ex
+# Copyright (C) 2024 Jakub Jelen <jjelen@redhat.com>
+# SPDX-License-Identifier: Apache-2.0
+#
+
+export KRYOPTIC_CONF="${TMPPDIR}/kryoptic.conf"
+cat >"${KRYOPTIC_CONF}" <<_EOF
+[[slots]]
+slot = 42
+dbtype = "nssdb"
+dbargs = "configDir='${TOKDIR}' flags='passwordRequired'"
+description = "Kryoptic Soft Token"
+_EOF
+
+# this overrides what we define in the generic init
+# the NSS DB can not store custom labels
+export TOKENLABEL="Kryoptic Soft Token"
```

Comment:
You can pass in some of these labels via configuration like you did for the passwordRequired Flags

Reply:
I wanted to make sure that we will really work with the NSS DB so using different one than the SQL DB. I am not sure if these particular knobs are used, but I can double-check

Reply:
Tokenlabel is needed by opensc pkcs11-tool to be able to find the right slot/token for some of the operations.
I also do not think they should be identical to the std kryoptic run, I would see it specific like "Kryoptic NSSDB Token"

Reply:
My intention was to keep the most of the options default.

The `flags='passwordRequired'` is needed for gnutls p11tool to attempt to login when searching for private keys. Without that, it skips the login and fails. It might make sense to describe this in the comment in the script rather than here though.